### ISSUE TITLE
Training Loss

### ISSUE THREAD

**Issue by NOP1224** [muzairkhattak/ProText#11]
Thanks for your excellent project, I'm trying to reproduce your code on other datasets, could you please tell me approximately how much the loss drops to on datasets such as caltech101, Imagenet etc.

**Comment by NOP1224** [comment]
My losses drop to about 0.022 and then they stop dropping

**Comment by muzairkhattak** [comment]
Hi @NOP1224,

Thank you for showing interest in ProText! I apologize for the delayed response. 

Regarding your question, yes the training loss magnitude converges to a constant value depending on the dataset. This behavior is expected as the prompt embedding are limited towards distilling the LLM generated text data.

**Comment by NOP1224** [comment]
Hello, **Protext** is an excellent model.  When training **Protext** on my own dataset for the **OVMLR** task, I noticed its performance is significantly lower than **CuPL** and **CLIP-Zeroshot**. To which parameters is **Protext** particularly sensitive?
